### PR TITLE
Add code for RTDSK0 pneumatic actuator controls

### PR DESCRIPTION
**Pull request by jyin999**

### CODE REVIEW

**Comment by ZLLentz** [comment]
A few things:
1. I get the following warnings when I try to build locally. Are these a concern?
image
2. The project seems to be slightly messed up and searching for a reference to `GVLs/RTDSL0.TcGVL`that does not exist. Is this a file that was supposed to be included? I imagine that it is not, because this is RTDSK0.
3. I would like to push fixes for other things in the Travis CI checker once you are 100% done here, so let me know when you are 100% done.

**Comment by jyin999** [comment]
It seems all wrong.  Let me check and will get back to you soon.

**Comment by jyin999** [comment]
Hi Zach,

What the warning means?  The variable related inputs and outputs are linked to hardware IOs.  Plcprog is not working. But I tested these links last night.   I have very sample change on code.  If my current branch messed up the project, I can delete it.  I can fork your project again and copy my code over. Please let me know if it is correct way to try.

Thanks,
Jing

**Comment by ZLLentz** [comment]
Hi Jing, I could not track down where the warnings were coming from. My assumption was that a file was missing in the commit. But if every needed file is committed to git and the code is working for you, then we can merge and try to track it down later.

I'm going to do a quick check to see if these issues already existed before you added code to make sure I'm not confusing things.

If you have further changes it doesn't matter if you push to this branch or make a new branch, do what is most convenient for you.

File: plc-kfe-motion/kfe_motion/POUs/PRG_RTDSK0.TcPOU [hunk 1]
```diff
@@ -3,26 +3,99 @@
...
-    fbMotion4: FB_MotionStageSim;
-    fbMotion5: FB_MotionStageSim;
-    fbMotion6: FB_MotionStageSim;
-    fbMotion7: FB_MotionStageSim;
-    fbMotion8: FB_MotionStageSim;
+	
+   {attribute 'TcLinkTo' := '.i_xInsertedLS := TIIB[RTDSK0-MPA-EL1008]^Channel 1^Input;
+							 .i_xRetractedLS := TIIB[RTDSK0-MPA-EL1008]^Channel 2^Input;
+							 .q_xInsert_DO :=  TIIB[RTDSK0-MPA-EL2008]^Channel 1^Output;
+							 .q_xRetract_DO :=  TIIB[RTDSK0-MPA-EL2008]^Channel 2^Output;
```
Comment (ZLLentz) [inline plc-kfe-motion/kfe_motion/POUs/PRG_RTDSK0.TcPOU:10]: Hi Jing, I figured out the link warning. The final line of these `TcLinkTo` pragmas are not supposed to end with a semicolon. See example at https://github.com/pcdshub/lcls-plc-kfe-motion/blob/master/plc-kfe-motion/kfe_motion/GVLs/Main.TcGVL#L31. If you remove the last semicolon in each pragma the warnings go away.
```suggestion
							 .q_xRetract_DO :=  TIIB[RTDSK0-MPA-EL2008]^Channel 2^Output
```

**Pull request closed without merging**